<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalendarDialog</class>
 <widget class="QDialog" name="CalendarDialog">
  <property name="windowTitle">
   <string>Выбор периода блокировки</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="instructionLabel">
     <property name="text">
      <string>Выберите дату начала и конца блокировки:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="startLayout">
       <item>
        <widget class="QLabel" name="startLabel">
         <property name="text">
          <string>Дата начала:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCalendarWidget" name="startCalendar"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="endLayout">
       <item>
        <widget class="QLabel" name="endLabel">
         <property name="text">
          <string>Дата конца:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCalendarWidget" name="endCalendar"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
